docs/resource-guide.md

```diff
@@ -99,6 +99,14 @@ This returns a populated resource struct as a `Res`. It shouldn't populate any
 values which already have the correct default as the golang zero value. In
 general it is preferable if the zero values make for the correct defaults.
 
+It should be noted that the default values are not expected to form a resource
+that is sensible or functional. Only fill values for which the zero value
+makes no sense (e.g., the `state` parameter of the `file` resource).
```

Comment:
As alluded to in my earlier comment, I don't think this part of this particular sentence is correct.

Reply:
"Generally, you will want to fill values..." ?

Reply:
I don't want to write it for you... ;) I was hoping my super long comment in https://github.com/purpleidea/mgmt/pull/481#issuecomment-468481155 was helpful enough to give you the knowledge to be able to rephrase into "human" since I suck at docs :(